appinventor/components/src/com/google/appinventor/components/runtime/Navigation.java

```diff
@@ -0,0 +1,352 @@
+package com.google.appinventor.components.runtime;
+
+import org.osmdroid.util.GeoPoint;
```

Comment:
You should clean up the imports following the [Google Java Style guide](https://google.github.io/styleguide/javaguide.html#s3.3-import-statements).